service/confirm-reminder/src/index.ts

```diff
@@ -15,12 +17,12 @@ const queueConfirm = process.env.CONFIRM_QUEUE || args.qc || "";
 const queueConfirmed = process.env.CONFIRMED_QUEUE || args.qd || "";
 const emailQueue = process.env.EMAIL_QUEUE || args.qe || "";
 const remindExchange = process.env.REMIND_EXCHANGE || args.qe || "";
-const maxRetries = parseInt(process.env.MAX_RETIRES || args.r || "3");
+const maxRetries = parseInt(process.env.MAX_RETRIES || args.r || "3");
+const maxPeriod = parseInt(process.env.MAX_RETRY_PERIOD || "5");
 const retryArray = (process.env.RETRY_INTERVAL || "2,3").split(",").map(x => parseInt(x)).filter(x => x > 0);
```

Comment:
I don't get why maxPeriod is an external param, can't it be generated automatically with 

`const maxPeriod = retryArray.reduce ( (max, d) => (max+d),0)

last but not the least (and not directly linked to this PR:

I don't get the link between maxRetries and retryPeriod. MaxRetries = 3 but only 2 numbers into the retry array. What is happening to the 3rd retry? when is this triggered?

ie. shouldn't maxRetries be retryArray.length?

Reply:
With retries, we are counting all emails sent to supporters, including the first one (when a supporter takes action) which is not exactly a retry.
So, we can use retryArray.length + 1 for maxRetries, if we are sure we will never want to retry more times than that.
Also, we can calculate maxPeriod from retryArray, I see no good reason not to remove two external params.
I'll remove.